### ISSUE TITLE
Couldn't find a suitable function named List with args ():

### ISSUE THREAD

**Issue by Ivo-Balbaert** [ValeLang/Vale#94]
On the first page of the Guide: (https://vale.dev/guide/introduction) $ Lists,
the code snippet:

fn main() export {
  l = List<Int>();
  l.add(1);
  l.add(3);
  l.add(7);
  println(l);
}

gives the compiler error:  Couldn't find a suitable function named List with args (): l = List<Int>();

ufcs.vale:2:8: Couldn't find a suitable function named split with args (str, str):

**Comment by Verdagon** [comment]
Seems there were two problems: List<Int> instead of List<int>, and we never implemented println of a list. I think I'll take out the println of list until we have our metaprogramming implemented.

`cat vstl/list.vale /dev/stdin | curl -X POST https://us-central1-valesite.cloudfunctions.net/vale-build -d @-` with this stdin:
```
fn main() export {
  l = List<int>();
  l.add(1);
  l.add(3);
  l.add(7);
  println(l.get(0));
}
```

works nicely. Will mark this fixed when that update is pushed to the site.

**Comment by Verdagon** [comment]
Fixed in v0.1.1. Thanks Ivo!
